test/e2e/internalencryption/README.md

```diff
@@ -0,0 +1,15 @@
+# Internal Encryption E2E Tests
+
+In order to test Internal Encryption, this test looks at `tls_connection_count` metrics from the Activator and QueueProxy.
+
+The `metricsreader` test image [here](../../test_images/metricsreader/README.md) was created for this purpose. Given the PodIPs of the Activator and the Knative Service pod (i.e. the QueueProxy), it will make requests to each respective `/metrics` endpoint, pull out the `*_tls_connection_count` metric, and respond with the value.
```

Comment:
Can `metricsreader` find the pods using label queries and/or DNS queries (if the services are [headless services](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services) -- I don't know if / how that would perturb the rest of Knative, though.)

Reply:
Hmm I'm not sure I'm following. Are you suggesting that the metricsreader image itself gets the IPs, rather than gathering that info as part of the test code?

Right now, I have the test getting that info, since it seemed easier to test and make sure everything was up and ready. My initial intention was to make the metricsreader image do as little as possible.